src/commonMain/kotlin/com/charleskorn/kaml/YamlOutput.kt

```diff
@@ -93,7 +93,11 @@ internal class YamlOutput(
         }
     }
 
-    override fun encodeEnum(enumDescriptor: SerialDescriptor, index: Int) = emitQuotedScalar(enumDescriptor.getElementName(index), configuration.singleLineStringStyle.scalarStyle)
+    override fun encodeEnum(enumDescriptor: SerialDescriptor, index: Int) {
+        val element = enumDescriptor.getElementName(index)
+
+        emitQuotedScalar(element, configuration.singleLineStringStyle.scalarStyle)
+    }
```

Comment:
note: I think if we don't change the encoding we should revert the code to the original state

Reply:
Its the same code, rather descriptive.